### PR TITLE
Use isXInADBrain booleans

### PR DESCRIPTION
**Pull request by JessterB**
* AG-263: Display new booleans in  gene network sidepanel & similar genes table
* AG-290: Replace all instances of legacy isChangedInADBrain with new isAnyRNAChangedInADBrain

### CODE REVIEW

File: src/app/core/services/gene.service.spec.ts [hunk 1]
```diff
@@ -43,16 +43,17 @@ describe('Service: Gene: TestBed', () => {
...
+    // it('should update the previous gene with the current gene', () => {
+    //     const upgSpy = spyOn(geneService, 'updatePreviousGene').and.callThrough();
+    //     geneService.setCurrentGene(mockGene1);
+    //     expect(geneService.getCurrentGene()).toEqual(mockGene1);
+    //     expect(geneService.getPreviousGene()).toEqual(undefined);
+    //
+    //     geneService.updatePreviousGene();
+    //     expect(upgSpy).toHaveBeenCalled();
+    //     expect(geneService.getPreviousGene()).toEqual(mockGene1);
+    // });
```
Comment (mfazza) [inline src/app/core/services/gene.service.spec.ts]: Why not just delete this?

File: src/app/genes/gene-similar/gene-similar.component.ts [hunk 2]
```diff
@@ -302,8 +305,8 @@ export class GeneSimilarComponent implements OnInit {
                     return this.druggability(rowObj.druggability);
                 }
             } else {
-                if (field === 'isIGAP' || field === 'isChangedInADBrain' ||
-                    field === 'haseqtl') {
+                if (field === 'isIGAP' || field === 'isAnyRNAChangedInADBrain' ||
+                    field === 'isAnyPRoteinChangedInADBrain' || field === 'haseqtl') {
```
Comment (mfazza) [inline src/app/genes/gene-similar/gene-similar.component.ts]: Oh, that's what Anna was talking about yesterday

**Review by mfazza — APPROVED** [review]
💯 🥇

File: src/app/core/services/gene.service.spec.ts (hunk 1 above)
Comment (JessterB) [reply src/app/core/services/gene.service.spec.ts]: I can move the test case into the related JIRA ticket if having it in here makes you sad!

**Comment by JessterB** [comment]
Found a bug, putting this into draft state while I sort it out...

File: src/app/core/services/gene.service.spec.ts (hunk 1 above)
Comment (JessterB) [reply src/app/core/services/gene.service.spec.ts]: I used the built-in 'xit' feature to suppress the test from execution, so this is better now.

**Pull request merged**